xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-test/xwiki-platform-extension-test-pageobjects/src/main/java/org/xwiki/extension/test/po/DependencyPane.java

```diff
@@ -92,7 +92,7 @@ public String getName()
         }
         // Unknown dependency, with no link.
         String innerText = container.getText();
-        return innerText.substring(0, innerText.indexOf(getVersion()));
+        return innerText.substring(0, innerText.indexOf(getVersion())).trim();
```

Comment:
This feels overly complicated. Maybe there was no ``extension-name`` wrapper when this method was written but we have it now so we should get the extension name from there. This method should have a single line, like the next one (``getVersion``). Can you make this change?

Reply:
Indeed, updated in the latest commit